TLM/TLM/UI/SubTools/LaneArrowTool.cs

```diff
@@ -128,6 +128,13 @@ public LaneArrowTool(TrafficManagerTool mainTool)
             }
 #endif
             ExtSegmentEndManager segEndMan = ExtSegmentEndManager.Instance;
+            int segmentEndId = segEndMan.GetIndex(segmentId, nodeId);
+            if (segmentEndId <0) {
+#if DEBUG
+                Debug.LogError($"Node {nodeId} is not connected to segment {segmentId}");
```

Comment:
Isn't there our own logging facility in TM:PE?

Reply:
Yes, I can replace it :wink: